visualization/app/codeCharta/ui/loadingGif/loadingGif.service.ts

```diff
@@ -25,6 +25,14 @@ export class LoadingGifService {
 		this.notifyLoadingMapFlagChange()
 	}
 
+	public isLoadingNewFile(): boolean {
```

Comment:
as mentioned above, i am not sure if this belongs here conceptually. The UI component responsible for showing some sort of loading indicator should not be responsible for felling other components if we are loading a map. Don't we have another way of knowing if a map is being loaded?